api-tester/src/main/java/com/revenuecat/apitester/kotlin/PurchasesAPI.kt

```diff
@@ -69,12 +69,22 @@ private class PurchasesAPI {
         }
         purchases.syncPurchases()
         purchases.getOfferings(receiveOfferingsCallback)
-        purchases.getSubscriptionSkus(skus, skusResponseCallback)
-        purchases.getNonSubscriptionSkus(skus, skusResponseCallback)
+
+
+//        purchases.getSubscriptionSkus(skus, skusResponseCallback)
+//        purchases.getNonSubscriptionSkus(skus, skusResponseCallback)
+        purchases.getProducts(productIds, productsResponseCallback)
+
+        // we need these for hybrids... these all fall back on some "best offer" or just purchase the base plan
         purchases.purchaseProduct(activity, storeProduct, upgradeInfo, purchaseChangeCallback)
         purchases.purchaseProduct(activity, storeProduct, purchaseCallback)
         purchases.purchasePackage(activity, packageToPurchase, upgradeInfo, purchaseChangeCallback)
         purchases.purchasePackage(activity, packageToPurchase, purchaseCallback)
+
+        // new methods -- pull desired purchaseoption from the storeproduct, doesn't have to be package vs product anymore
+        purchases.purchase(activity, purchaseOption, upgradeInfo, purchaseChangeCallback)
```

Comment:
From Maddie: either we have to add purchase options to BC4 products OR we need to only have this new api for BC5 somehow

Reply:
For unityIAP we do want to use both hte new API and BC4, right? I think it should be fairly easy to support this, right?
https://revenuecats.atlassian.net/browse/CF-978

Reply:
i started down that path, but i think forcing users to use a PurchaseOption in BC4 is a littel funny. i'll try it out though and we can evaluate